src/mongoose_user_stats.erl

```diff
@@ -0,0 +1,76 @@
+-module(mongoose_user_stats).
+
+-author('aleksander.lisiecki@erlang-solutions.com').
+
+% TODO replace tid to official ESL one
+-define(URL_BASE, "https://www.google-analytics.com/collect?v=1&tid=UA-151110014-1&t=event").
+
+-export([report_user_stats/0]).
+
+report_user_stats() ->
+    IsAllowed = ejabberd_config:get_local_option_or_default(is_report_user_stats_allowed, true),
```

Comment:
By default reporting should be `disabled`, we change it to `enabled` when we think this functionality is ready.
Also I'm wondering if name like `report_user_stats` with possible values `enabled`, `disabled` would be better here. What do you think?

Reply:
Yes, finally I want it to be enabled by default. Currently this is not ready to be used by our community, that's why I want it to be disabled by default. When we decide it's ready to use in real world, we'll change to enabled by default.